website/admin.py

```
@@ -1271,3 +1275,60 @@ class UserTaskSubmissionAdmin(admin.ModelAdmin):
+        if self.value() == "active":
+            return queryset.exclude(last_login__isnull=True)
+        if self.value() == "inactive":
+            return queryset.filter(last_login__isnull=True)
+        return queryset
+
+
+admin.site.unregister(User)
+
+
+class CustomUserAdmin(DjangoUserAdmin):
+    actions = [deactivate_users]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)